### ISSUE TITLE
pandas or pandas_market_calendar bug?

### ISSUE THREAD

**Issue by gpernelle** [rsheftel/pandas_market_calendars#110]
Hello, we recently encountered a strange behavior, and we tracked the problem to having different pandas versions:
with pandas==1.1.0 the following code returns false

```python 
import pandas as pd
import pandas_market_calendars as mcal  
nyse = mcal.get_calendar('NYSE')       
early = nyse.schedule(start_date='2012-07-01', end_date='2012-07-10') 
nyse.open_at_time(early, pd.Timestamp('2012-07-03 12:00', tz='America/New_York'))    
>>> False
```

The same code with pandas==1.0.3 returns True

**Comment by incredibleray** [comment]
Encountering the same issue, nyse.open_at_time(...) returns False all the time regardless of what timestamp I input.

**Comment by rsheftel** [comment]
Thank you for this error report. This is fixed in commit 9b9278d18b4f1b214b765a3a20730df4f63176b0 and on PyPi as v1.4.2

**Comment by incredibleray** [comment]
thank you for the very timely fix 👍 💯
